contrib/forms/psychiatrySet/brief_aan_verwijzer/new.php

```diff
@@ -146,27 +148,19 @@ function getPatientDateOfLastEncounter($nPid)
 
 <html>
 <head>
-    <link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker/build/jquery.datetimepicker.min.css">
+    <?php Header::setupHeader('datetime-picker'); ?>
 </head>
 
-
-
 <body <?php echo $top_bg_line;?> topmargin=0 rightmargin=0 leftmargin=2 bottommargin=0 marginwidth=2 marginheight=0>
 
 <style type="text/css">
  body       { font-family:sans-serif; font-size:10pt; font-weight:normal }
-  .dehead    { color:#000000; font-family:sans-serif; font-size:10pt; font-weight:bold;
+  .dehead    { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:bold;
                 padding-left:3px; padding-right:3px; }
-                 .detail    { color:#000000; font-family:sans-serif; font-size:10pt; font-weight:normal;
+                 .detail    { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:normal;
                                padding-left:3px; padding-right:3px; }
 </style>
 
-<script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../../library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
```

Comment:
Testing: Why was this removed?

Reply:
This is included in all calls to  Header::setupHeader